src/v/cluster/partition_manager.cc

```diff
@@ -11,6 +11,7 @@
 
 #include "cluster/logger.h"
```

Comment:
```
application: do not shutdown internal partitions
During redpanda shutdown sequence we stop all partitions to prevent
kafka requests from waiting before shutting down the application.
Internal topics are managed on their own and they do not have to be shut
down early.
```

this commit message doesn't state why we want to do this. is it fixing a bug? if it isn't fixing something or making something significantly better let's move it to a new PR.